ibm/data_source_ibm_is_endpoint_gateway_targets.go

```diff
@@ -157,9 +157,38 @@ func dataSourceIBMISEndpointGatewayTargetsRead(context context.Context, d *schem
 			}
 			resourceInfo = append(resourceInfo, l)
 		}
-		d.Set(isVPEResources, resourceInfo)
-		d.SetId(dataSourceIBMISEndpointGatewayTargetsId(d))
+		staticService := map[string]interface{}{}
+		staticService[isVPEResourceName] = "ibm-ntp-server"
+		staticService[isVPEResourceType] = "provider_infrastructure_service"
+		resourceInfo = append(resourceInfo, staticService)
+	}
+	queryString := "doc.extensions.virtual_private_endpoints.endpoints.ip_address:*"
+	fields := []string{"name", "region", "family", "type", "crn", "tags", "organization_guid", "doc.extensions", "doc.resource_group_id", "doc.space_guid", "resource_id"}
+	getSearchOptions := globalsearchv2.SearchBody{
+		Query:  queryString,
+		Fields: fields,
+	}
+	globalSearchClient, err := meta.(ClientSession).GlobalSearchAPI()
+	if err != nil {
+		return diag.FromErr(err)
 	}
+	searchResult, err := globalSearchClient.Searches().PostQuery(getSearchOptions)
+	if err != nil {
+		log.Printf("[DEBUG] PostQuery on globalSearchApi failed %s", err)
```

Comment:
Lets specify the operation where this call is being made - Customer won't check into the code to understand the error `PostQuery on globalSearchApi retrieving endpoint gateway targets failed`

Reply:
Added query string in the log